atomicfu/src/commonMain/kotlin/kotlinx/atomicfu/MangledJsNames.kt

```diff
@@ -0,0 +1,63 @@
+package kotlinx.atomicfu
+
+/**
+ * All atomicfu declarations are annotated with [@JsName][kotlin.js.JsName] to have specific names in JS output.
+ * JS output transformer relies on these mangled names to erase all atomicfu references.
+ */
+
+// Atomic factory functions
+internal const val ATOMIC_REF_FACTORY = "atomic\$ref\$"
+internal const val ATOMIC_REF_FACTORY_BINARY_COMPATIBILITY = "atomic\$ref\$1"
+internal const val ATOMIC_INT_FACTORY = "atomic\$int\$"
+internal const val ATOMIC_INT_FACTORY_BINARY_COMPATIBILITY = "atomic\$int\$1"
+internal const val ATOMIC_LONG_FACTORY = "atomic\$long\$"
+internal const val ATOMIC_LONG_FACTORY_BINARY_COMPATIBILITY = "atomic\$long\$1"
+internal const val ATOMIC_BOOLEAN_FACTORY = "atomic\$boolean\$"
+internal const val ATOMIC_BOOLEAN_FACTORY_BINARY_COMPATIBILITY = "atomic\$boolean\$1"
+
+// Atomic value
+internal const val ATOMIC_VALUE = "kotlinx\$atomicfu\$value"
+
+// Atomic operations
+internal const val COMPARE_AND_SET = "compareAndSet\$atomicfu\$"
+internal const val GET_AND_SET = "getAndSet\$atomicfu\$"
+internal const val GET_AND_INCREMENT = "getAndIncrement\$atomicfu\$"
```

Comment:
could you please elaborate on why `atomicfu` substring is inserted in the middle of the signature?

I do not really mind, it just looks a bit alien and a more familiar `atomicfu\$getAndIncrement` can be used instead

Reply:
Actually, this turned out to be just a crutch for JS transformer that is no longer needed. So, I also fixed all other mangled names to have `atomicfu$` prefix for consistency.